docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -59,11 +59,21 @@ to the given repositories, applicability data is generated for them as well.
 Generated applicability data can be queried using 
 the `Query Content Applicability` API described below.
 
-The API will return a :ref:`group_call_report`. Users can check whether the applicability
-generation is completed using group id in the :ref:`group_call_report`. The `_href` in the
-:ref:`group_call_report` will point to the root of `task_group` resource. However, this API
-endpoint currently returns 404 in all cases. You can append '/state-summary/' to the URL and
-perform a GET request to retrieve a :ref:`task_group_summary`.
+There are two ways to regenerate applicability for consumers bound to a repo. The computation can
+be done linearly or in parallel. An optional `parallel` argument determines which method is used.
+
+When computing applicability linearly, the API will return a :ref:`call_report` when parallel is
+False or not present. In this case, users can check whether the applicability generation is
+completed using task id in the :ref:`call_report`. You can run a single applicability generation
+task at a time. If an applicability generation task is running, any new applicability generation
+tasks requested are queued and postponed until the current task is completed.
+
+When computing applicability in parallel, the API will return a :ref:`group_call_report`. Users can
+check whether the applicability generation is completed using group id in the
```

Comment:
Add backticks around `group id`. Also consider rewriting s/using group id/using the `group id` field/